tests/StackExchange.Redis.Tests/Sentinel.cs

```diff
@@ -12,7 +12,7 @@ namespace StackExchange.Redis.Tests
     public class Sentinel : TestBase
     {
         private string ServiceName => TestConfig.Current.SentinelSeviceName;
-        private ConfigurationOptions ServiceOptions => new ConfigurationOptions { ServiceName = ServiceName, AllowAdmin = true };
+        private ConfigurationOptions ServiceOptions => new ConfigurationOptions { ServiceName = ServiceName, AllowAdmin = true, Password = "changeme" };
```

Comment:
This is commented in config and present here - should be removed here I think, guessing it was during testing?

Reply:
Didn’t seem to affect anything negatively to leave it. Again, just say so an I will remove them all.